### DISCUSSION TITLE
Is it possible to integrate the SFMC dev tool with Jenkins application

### DISCUSSION THREAD

**Discussion by ajaysharma7884** [Accenture/sfmc-devtools#888] [Q&A]
@JoernBerkefeld Could you please provide some suggestion to integrate the SFMC dev tool github with Jenkins application. I have tried to integrate my SFMC dev tool through connecting with GitHUB with Jenkins application, but I am getting issue in Jenkins application build log that it have an issue in mcdev init. Kindly find the screenshot attached for your reference.

![failed-log](https://user-images.githubusercontent.com/117371684/235700040-fe4884d1-8b76-4a66-aaaf-691fa81f9560.PNG)

**Comment by JoernBerkefeld** [comment]
hi @ajaysharma7884 - i cannot support jenkins but reading that log i think im seeing an error about jenkins not finding `node`?
that's basically before mcdev is even executed.

**Reply by ajaysharma7884** [reply]
> hi @ajaysharma7884 - i cannot support jenkins but reading that log i think im seeing an error about jenkins not finding `node`? that's basically before mcdev is even executed.

@JoernBerkefeld We are trying to integrate our SFMC dev tool Github with jenkins, but while integrating will found that issue, due that it is not allowing us to process. Any suggestion if you have for mcdev we have missed in Jenkins app to implement??

**Reply by JoernBerkefeld** [reply]
Unfortunately, I have never configured Jenkins --> I don't know how to fix or rewrite this. Is anyone out there in the community that wants to help?
